src/tools/PrizeTierController/utils/formatPrettyPercentage.ts

```diff
@@ -1,8 +1,11 @@
 import { DPR_DECIMALS } from '@prizeTierController/config'
 
-export const formatPrettyPercentage = (value: number) => {
-  return (value / 10 ** (DPR_DECIMALS + 2)).toLocaleString('en', {
+export const formatPrettyPercentage = (
```

Comment:
this is specific to formatting dpr so I'd change the name to reflect that. Would also prefer sticking to using `formatNumberForDisplay` from our utils package and just passing the `style`

Reply:
Fixed through 0863a8a